addons/website_slides/tests/test_gamification_karma.py

```diff
@@ -148,14 +146,14 @@ def test_karma_gain_multiple_course_multiple_users(self):
             (self.channel | self.channel_2)._remove_membership(users.partner_id.ids)
 
         for user in users:
-            self.assertEqual(user.karma, 0)
```

Comment:
Same here, would complete the line above:
`# now, remove the membership in batch, on multiple users`
With an explanation that the karma should not move.